src/index.html

```diff
@@ -165,7 +165,7 @@ <h2>Combat Location [ no influence to the actual gameplay ]</h2>
 				<div id="start"><button type="submit"></button><span>Start</span></div>
 			</form>
 			
-			<div id="fullScreen"><a onclick="AncientBeast.requestFullscreen()"><div></div>Fullscreen</a></div>
+			<div id="fullScreen"><div></div><span class="fullscreen__title">Fullscreen</span></div>
```

Comment:
@TheSeally Is the double underline a thing?

Reply:
This is consistent with the BEM methodology. https://en.bem.info/

Reply:
Oh, seems like interesting stuff, thanks for sharing!

Reply:
Found the info about it over there, makes a lot of sense! https://en.bem.info/methodology/quick-start/#element

Reply:
btw I can change the name of the class if it is not suitable for the current repository